.github/workflows/run-tests.yml

```diff
@@ -2,12 +2,13 @@ name: Test Services
 
 on:
   push:
+    paths-ignore:
+      - '**.md'
+      - 'docs/**'
     branches-ignore:
       - master
-      - feat/game-demo
   pull_request:
     paths-ignore:
-      - 'Makefile'
       - '**.md'
       - 'docs/**'
```

Comment:
I tried to prevent the `run-tests.yml` to run when we only push changes to the docs. However it keeps getting triggered.

When I look online, I found https://github.com/actions/runner/issues/2324 which tell us that this condition is bug from github action.